ruby/occurrence_hash/lib/occurrence_hash.rb

```diff
@@ -0,0 +1,9 @@
+class CharOccurrenceChecker
+  def char_occurrence(string)
```

Comment:
1. Rename this method.
2. Do not use variable names like `string`, `hash` etc. Fix these naming issues in all exercises.
3. Variable names should be meaningful. Read: http://www.makinggoodsoftware.com/2009/05/04/71-tips-for-naming-variables/